src/vulnpy/trigger/xpath.py

```diff
@@ -0,0 +1,45 @@
+import lxml.etree
+import xml.etree.ElementTree as ET
+from io import StringIO
+
+xml = (
```

Comment:
could you make this a triple quoted string? it would be easier to read.

Also, it looks like py27 tests are failing because this must always be unicode. Changing it to
```
u"""
......
"""
```
should work